_projects/vulture.md

```diff
@@ -16,7 +16,8 @@ mentors:
 name: "Vulture dead code removal"
 requirements:
   - "The participant should have one patch to [vulture](https://github.com/jendrikseipp/vulture) accepted."
-  - "Being familiar with the current Bear API and what it provides. Fixing an issue for a Bear during the newcomer process would help with this."
+  - "Being familiar with the current Bear API and what it provides. Fixing an issue for a Bear during the newcomer
+	 process would help with this."
```

Comment:
For any line you need to change , please start additional lines with the first word under the first word of the first line.  i.e. "process" under "Being" here.

(dont modify extra lines which you dont need to modify ... only make the lines you modify better)